.prettierrc.json

```diff
@@ -1 +0,0 @@
-{}
```

Comment:
I thought `prettier` was not setup properly because of the empty configuration file and my Prrettier Extension misconfiguration so I thought it would be a good idea to follow what I thought were best practices being following from the `near-api-js` repo  and add eslint support. I think what makes most sense here is keep this PR at minimal changes and keep prettier's formatting functionality for now. 

I have added an issue to add proper eslint support with Turbo.

Reply:
Maybe the presence of `.prettierrc.json` might prompt IDEs to use the default prettier settings. But agree, let's add `eslint` in another PR and also add the `prettier-eslint` plugins. That's generally preferred as then `eslint` and `prettier` don't compete with eachother.